doc/services/object-store/objects.rst

```diff
@@ -495,9 +495,20 @@ filenames inside the archive.
 
 `Get the executable PHP script for this example <https://raw.githubusercontent.com/rackspace/php-opencloud/master/samples/ObjectStore/auto-extract-archive-files.php>`_
 
-
 Delete object
--------------
+---------------------------------
+
+.. code-block:: php
+
+  /** @var bool $deleted */
+  $deleted = $container->deleteObject('{objectName}');
```

Comment:
No need to have `$deleted =` or a type hint since this method doesn't return anything